### ISSUE TITLE
Some queries are not executable because of missing tables

### ISSUE THREAD

**Issue by facha** [cloudera/impala-tpcds-kit#9]
It would be nice to have a list of queries that can be executed. Some of them result in error because of missing tables.
E.g.

```
impala-shell -d tpcds_parquet -f q1.sql
ERROR: AnalysisException: Could not resolve table reference: 'store_returns'
```

**Comment by amosbird** [comment]
According to [this](https://github.com/cloudera/impala-tpcds-kit/blob/c899efba3358fdd1bb155a8c9afc908570b4c407/gen-facts.sh#L11), it's in the **TODO** list. Loading fact tables with manually tuned partition settings requires much more work than dim tables.

**Comment by ssavvides** [comment]
These are the supported queries according to [this](http://blog.cloudera.com/blog/2014/01/impala-performance-dbms-class-speed/)

Interactive
q19
q42
q52
q55
q63
q68
q73
q98

Reporting
q3
q7
q27
q43
q53
q89

Deep Analytic
q34
q46
q59
q79
ss_max

**Comment by gregrahn** [comment]
Fixed with 611fa0a425e76acaeb205e57c7279ecc4e517919
